src/modules/fancyzones/editor/FancyZonesEditor/GridResizer.xaml

```diff
@@ -6,6 +6,7 @@
              xmlns:local="clr-namespace:FancyZonesEditor"
              mc:Ignorable="d" 
              Focusable="True"
+             Initialized="ThumbInitialized"
```

Comment:
Can't set `AutomationProperties.Name` here directly?

Reply:
Thank you for the review. Addressed it in the latest commit. PTAL